kedro/framework/cli/starters.py

```diff
@@ -997,19 +1019,33 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         self.error_message = kwargs.get("error_message", "")
 
     def __str__(self) -> str:
+        # Format the title with optional color
         title = self.title.strip().title()
-        title = click.style(title + "\n" + "=" * len(title), bold=True)
-        prompt_lines = [title, self.text]
+        title = click.style(title, fg="cyan", bold=True)
+        title_line = "=" * len(self.title)
+        title_line = click.style(title_line, fg="cyan", bold=True)
+
+        # Format the main text
+        text = self.text.strip()
+
+        # Combine everything
+        prompt_lines = [title, title_line, text]
         prompt_text = "\n".join(str(line).strip() for line in prompt_lines)
         return f"\n{prompt_text}\n"
 
-    def validate(self, user_input: str) -> None:
-        """Validate a given prompt value against the regex validator"""
-        if self.regexp and not re.match(self.regexp, user_input):
-            message = f"'{user_input}' is an invalid value for {(self.title).lower()}."
+    @property
+    def user_input(self) -> str:
+        return self._user_input
+
+    @user_input.setter
+    def user_input(self, input: str) -> None:
```

Comment:
I'm not sure it's clear enough now that this is doing validation 🤔  I have a slight preference to keeping the `validate()` method as before.

Reply:
Fair point, the other way is more explicit.